### PR TITLE
Update docs

### LINKED ISSUE #109: add/improve docs for caikit+tgis setup
#107 added a `caikit` image which relies on a separate `tgis` container. This also includes an example setup of the `ServingRuntime`/`InferenceService` manifest that can be deployed and tested, but the documentation is missing and/or outdated.

### PR DESCRIPTION
**Pull request by dtrifiro**
## Description

Starting to cleanup/update docs for the new caikit+tgis manifests. Marking this as a draft for now to start some discussions.

closes #109

### CODE REVIEW

File: demo/kserve/built-tip.md [hunk 1]
```diff
@@ -13,9 +16,9 @@ git clone https://huggingface.co/bigscience/bloom-560m
 2. Create a virtual environment with Python 3.9 and install `caikit-nlp`
 
 ```bash
-python3 -m venv -p python3.9 venv
```
Comment (Xaenalt) [inline demo/kserve/built-tip.md:16]: In general, we say python 3.9 because 3.11 had some issues, and caikit was written with 3.9 in mind

**Review by Xaenalt — CHANGES_REQUESTED** [review]
One small change, but looks great to me!

File: demo/kserve/built-tip.md (hunk 1 above)
Comment (dtrifiro) [reply demo/kserve/built-tip.md:16]: Can you provide links to any of these issues?

**Comment by dtrifiro** [comment]
@heyselbi I think we should discuss the future of the docs/manifests/script in `demo/kserve`

**Review by Xaenalt — APPROVED** [review]

**Pull request merged**